docs/insomnia/get-started/install.md

```diff
@@ -3,13 +3,11 @@ layout: article-detail
 title:  Install Insomnia
 ---
 
-<!-- for rework: revise this page like we normally would -->
-
 Insomnia is available on macOS, Windows, and Linux. If you haven't already downloaded Insomnia, visit the [Download Page](https://insomnia.rest/download).
 
 <nav>
   <div class="nav nav-tabs" id="nav-tab" role="tablist">
```

Comment:
For reviewers: I think tabs is still the way to go here rather than sections for each (like https://learning.postman.com/docs/getting-started/installation/installation-and-updates) or a table of options (like https://docs.konghq.com/gateway/3.7.x/install/). I decided to not do the table of options because I don't think the install is different for EE or OSS and because all the install instructions can live on one page instead of separate pages.